tests/writers.rs

```diff
@@ -87,16 +108,32 @@ impl World for WriterWorld {
 
             winning_writer_name: String::from(""),
             winning_writer_address: String::from(""),
+
+            anvil: Option::None, // Anvil is started and stopped in before/after hooks. Better way to do this?
         })
     }
 }
 
+
+#[tokio::main]
 // Test runner
-fn main() {
-    // You may choose any executor you like (`tokio`, `async-std`, etc.).
-    // You may even have an `async` main, it doesn't matter. The point is that
-    // Cucumber is composable. :)
-    futures::executor::block_on(WriterWorld::run("tests/features"));
+async fn main() -> eyre::Result<()> {
+
+    let world = WriterWorld::cucumber()
+        // Start a fresh anvil before each scenario
+        .before(move |_, _, _, world| {
+            async move {
+                world.anvil = Option::Some(Anvil::new().spawn());
+                let endpoint = world.anvil.as_ref().unwrap().endpoint();
+                println!("Anvil running at `{}`", endpoint);
+            }.boxed()
+        })
+        .run_and_exit("tests/features");
```

Comment:
Is this where we could re-define the dir? E.g., so we could have "test/unit" and "test/acceptance", or "test/forge" and "test/cucumber", or something similar. Won't block merging on it, just thinking of improving repo IA